inc/gravity-forms.php

```diff
@@ -50,9 +50,13 @@ function ( $form_string, $form ) {
 		// Style the submit button.
 		$form_string = str_replace( 'gform_button', 'nhsuk-button', $form_string );
 		// Style the next button.
-		$form_string = str_replace( 'gform_next_button button', 'nhsuk-button', $form_string );
+		$form_string = str_replace( 'gform_next_button gform-theme-button button', 'nhsuk-button', $form_string );
```

Comment:
possibly add comment to show this is for Gravity Forms 2.7+ for easier future maintenance. Likewise for line 56

Reply:
Done, thanks.